docs/operating-systems/course-tools.md

```diff
@@ -148,51 +143,52 @@ If you haven’t installed gatorgrade type this command:
 
 ### Go installation process:
 
-If you haven't installed go click the link and follow [the instructions](https://go.dev/dl/)
+If you haven't installed Go: [the instructions](https://go.dev/dl/)
 
 Make sure you click the right chip that your laptop has.
 
 If you choose the right version and it still isn't working, follow these instructions:
-1. type in the command ```nano ~/.zshrc``` into your terminal
-2. add in the following 2 paths into the file that pops up in your terminal:
+- Type in the command ```nano ~/.zshrc``` into your terminal
+- Add in the following 2 paths into the file that pops up in your terminal:
   - ```export PATH=$PATH:/usr/local/go/bin```
   - ```export PATH=$PATH:$GOPATH/bin```
-3. After you save the changes in that file, source the file using the command ```. ~/.zshrc```.
-4. Check if go is now working using the ```go version``` command.
+- After you save the changes in that file, source the file using the command ```. ~/.zshrc```.
+- heck if go is now working using the ```go version``` command.
 
 
 ### GCC installation process:
 
-GCC might already be installed on your computer since you have a mac. To check type:
+GCC might already be installed on your computer since you have a Mac. To check type:
 
 `gcc –version`
 
-You want your output to say you have the clang version.
-
-
-
+Your output should specify you have the clang version.
 
 ## Locally installing the required tools for Windows
 
-### gcc & ```make``` command
-
-1. Do not install MinGW directly, go to https://www.msys2.org/ and download MSYS2.
-2. Search for the MSYS2 MinGW x64 environment and open it.
-3. Write ```pacman -Syuu``` to update the environment.
-4. Write ```pacman -S mingw-w64-x86_64-toolchain``` to install the toolchain, which contains gcc and the make command.
-5. Verify the installation worked by entering ```gcc —version``` in your terminal. If it is working it should return the version of gcc you installed.
-6. In order to use the gcc toolchain, you have to add this to your path environment variable: <MSYS2 location>/mingw64/bin .
-7. You can also change the name of make from mingw32-make to make to make it easier to execute in the terminal.
-8. In order to test everything went correctly, create two new files in a directory named hello.c and makefile.
-    1. Hello.c should contain:
-      ``` #include <stdio.h>
+### GCC & `make` command
+- Do not install MinGW directly, go to https://www.msys2.org/ and download MSYS2.
```

Comment:
I would recommend you change this link to the format you used above in the different languages.